### ISSUE TITLE
Install instructions missing a step?

### ISSUE THREAD

**Issue by muff1nman** [airsonic/airsonic-ui#18]
Working on getting a dev instance running and ran into this issue:

```
$ ng serve
node_modules appears empty, you may need to run `npm install`
```
This was after I had installed angular according to the instructions. After doing an `npm install`, things seem to get along farther. Perhaps we can reduce the `npm install -g @angular/cli` to `npm install`?

**Comment by bfkelsey** [comment]
A global version of @angular/cli is required so just adding npm install to it should work.

**Comment by bfkelsey** [comment]
This has been fixed with #f61a6b4
